src/test/java/qwerty/command/DeadlineCommandTest.java

```diff
@@ -0,0 +1,104 @@
+package qwerty.command;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import qwerty.QwertyException;
+import qwerty.Storage;
+import qwerty.TaskList;
+import qwerty.Ui;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Maybe it would be good to expand the wildcard import to show only the methods that you need.

Reply:
Accidental move by IDE auto-import, will fix.